testsuite_how_to.md

```diff
@@ -0,0 +1,199 @@
+How to Run the Test Suite and Add Tests
+========================
+This page is meant to help familiarize a new user with using the test suite. It explains how to run the example Java test suite,
+and how to add more tests to the suite. 
+
+For an overview of the test suite, refer to [Test Suite Overview](testsuite_overview.md)
+
+How to Run the Example Java Suite
+--------------------
+The test suite itself is language-independent, and requires additional language support before tests are run on a Rest.li
+client implementation. 
+ 
+To validate that these tests all pass using the Java implementation of the Rest.li client, *and* to demonstrate how
+to use the test suite, this project contains a Java TestNG suite, located in the client-testsuite/src/test/java folder.
+
+To run the provided TestNG suite:
+
+```
+gradle test
+```
+
+When you run ```gradle test```, TestNG runs tests provided from two Java classes in the example test suite:
+* TestRestClientWithManualAssertions: manual assertion tests that have to be built by hand. 
+* TestRestClientAgainstStandardTestSuite: automated tests driven by manifest.json. Within each category of test (i.e. json,
+schema, or wire protocol), the same testing logic is shared, and testing data is assigned by manifest.json. 
+
+
+```gradle test``` will run successfully, since the Java implementation of the Rest.li client passes all the tests.  
+
+This is how to run the Java test suite, but because the build is successful, it's not immediately clear what is 
+happening when ```gradle test```  is run. For better illustration, this guide will help you make tests that fail 
+when run.
+
+
+##### Manual Assertion Tests
```

Comment:
I prefer to organize this section as follows:
-Data driven automated tests
-Manual Assertion Tests
-How to run them